### DISCUSSION TITLE
Disable suggestion box footers/headers

### DISCUSSION THREAD

**Discussion by rushiiMachine** [google/mozc#1135] [Ideas]
I would love a toggle for the suggestion box footers (or headers, I guess) like `Tabキーで選択` , it's pretty useless once you've used mozc for a while.

![2024-12-08_13-33-15](https://github.com/user-attachments/assets/483aad70-965d-4dea-9c4a-f1a83c2d5ec8)

They seem to be added here:
https://github.com/google/mozc/blob/e983ab86bb77b9ceb49f268fc824dc64e4ab87d8/src/session/internal/session_output.cc#L355-L401

**Comment by hiroyuki-komatsu** [comment]
Hi rushiiMachine,
Thanks for the screenshot. It's helpful for me to understand your situation.

It looks you use Fcitx. So I'd suggest you to talk with the Fcitx project team.
As the function name describes, that label is expected to be displayed on the footer.

![image](https://github.com/user-attachments/assets/e6ba66ee-2d08-49b8-b022-b6a743d8fd1f)

**Reply by wengxt** [reply]
Regardless it's at top or bottom in fcitx, is it Ok to not display this field at all, or is there any critical usage of this field that may affect usability?

**Reply by rushiiMachine** [reply]
They seem to be just shortcut hints -- I don't think there's anything critical about them.

I created a [discussion](https://github.com/fcitx/fcitx5/discussions/1205) in the fcitx5 repo!

**Reply by hiroyuki-komatsu** [reply]
So far, the footer label is used as a supplemental information, but it is designed to be visible to the user.

My suggestion is to display it to the footer area.
